packages/Toast.vue

```diff
@@ -109,37 +109,51 @@
           </div>
         </template>
       </div>
-      <template v-if="toast.cancel">
-        <button
-          :class="cn(classes?.cancelButton, toast.classes?.cancelButton)"
-          data-button
-          data-cancel
-          @click="
-            () => {
-              deleteToast()
-              if (toast.cancel?.onClick) {
-                toast.cancel.onClick()
-              }
-            }
-          "
-        >
-          {{ toast.cancel.label }}
-        </button>
-      </template>
-      <template v-if="toast.action">
-        <button
-          :class="cn(classes?.actionButton, toast.classes?.actionButton)"
-          data-button
-          @click="
-            (event) => {
-              toast.action?.onClick(event)
-              if (event.defaultPrevented) return
-              deleteToast()
-            }
-          "
-        >
-          {{ toast.action.label }}
-        </button>
+
+      <template v-if="toast.cancel || toast.action">
+        <div data-buttons="">
```

Comment:
The `template` wrapper is unnecessary. We can have the directive `v-if="toast.cancel || toast.action"` on the `div` element straightaway, being the only element within.